media/js/src/multipleChoiceQuestion2.jsx

```diff
@@ -96,30 +96,30 @@ export const MultipleChoiceQuestion2 = ({isSubmitted, setIsSubmitted, takeaways,
                 ))
             }
             {!nextStep &&
-                <button className="btn btn-secondary me-2"
+                <button className="btn btn-sm btn-success mt-3"
```

Comment:
You might want to provide a margin between these buttons for when they appear together. They look a little smooshed here: 
![Screenshot 2024-09-23 at 4 27 34 PM](https://github.com/user-attachments/assets/4b1c66b5-57a2-47ff-a4b9-9b55101aa93b)

Reply:
I'll probably put it back at some point, I just need to see things in multiple device sizes for the time being.